File: wrappers/mvc/demos/KendoUI.Mvc.Web.Examples/Controllers/HomeController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using KendoUI.Mvc.Web.Examples.Models;

namespace KendoUI.Mvc.Web.Examples.Controllers
{
    public class HomeController : Controller
    {
        public ActionResult Index()
        {
            return RedirectToRoute("RazorDefault", new { controller = "Suite" });
        }
    }
}

File: wrappers/java/spring-demos/src/main/java/com/kendoui/spring/controllers/treeview/HomeController.java
package com.kendoui.spring.controllers.treeview;

import java.util.Locale;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;

import com.kendoui.spring.models.ProductDao;

@Controller("treeview-home-controller")
@RequestMapping(value="/web/treeview/")
public class HomeController {
    @RequestMapping(value = {"/", "/index"}, method = RequestMethod.GET)
    public String index(Locale locale, Model model) {
        return "web/treeview/index";
    }
}

